src/components/utils/string.js

```diff
@@ -28,3 +28,19 @@ export const startCase = str => asWordsList(str).map(firstUpper).join(' ')
 export const humanCase = str => firstUpper(asWordsList(str).join(' '))
 
 export const randomId = () => Math.random().toString(36).substring(7)
+
+export const includesRegex = (string, regex) => !!string.match(regex)
+
+/*
+Tested pass against:
+https://api.onfido.com/v2/documents
+https://onfido.com/v2/documents
+https://cross.onfido.com/v2/documents
+https://cross.lol.onfido.com/v2/documents
+
+Tested fail against:
+https://revolut.com/v2/documents/?url=https://onfido.com", /https:\/\/[A-Za-z0-9\.]*\.?onfido\.com/g
+https://onfido.revolut.com/v2/documents
+https://onfido.revolut.com/v2/documents/onfido.com
+ */
+export const isOnfidoURL = (url) => includesRegex(url,/^https:\/\/[A-Za-z0-9\.]*\.?onfido\.com/g)
```

Comment:
Good point, end anchor be enough? 

```
/^https:\/\/[A-Za-z0-9\.]*\.?onfido\.com$/g
```

Reply:
End anchor's not going to let us have `https://onfido.com/myscript.js` or anything
It might be worth going for more of a solution of actually URL-parsing

[There are a couple of hacks you can do](https://stackoverflow.com/questions/736513/how-do-i-parse-a-url-into-hostname-and-path-in-javascript), or you can use the `URL` constructor (not supported by IE11)

Or you can go for a much more involved regex... But it won't be readable 😄

Reply:
This only checks against `window.location.hostname`, right? It seems to me there is no need to match pathname too

Reply:
At the moment - yeah, you're right.

If we go for that solution, then - I'd just say we should fix up the method name, from `isOnfidoURL` to `isOnfidoHostname` or something, to emphasise that it doesn't work with a generic URL.

Although I'd still rather a more general solution, imo we'll probably want it to work with paths at some point

Reply:
I could do `export const isOnfidoURL = (url) => new URL(url).hostname.includes("onfido")`

Reply:
I'm not sure if we want it to work with paths at some point. At the moment we just want to check if the breadcrumb was an xhr on an onfido domain url. I think I will rename the function to `isOnfidoHostname` and use the current regex.

Reply:
Cool - in that case I'd just say go with what @elhesuu was saying, and add a `$` to the end of the regex :)